presto-main/src/main/java/com/facebook/presto/operator/PartitionedOutputOperator.java

```diff
@@ -351,9 +351,11 @@ public PagePartitioner(
         // Does not include size of SharedBuffer
         public long getRetainedSizeInBytes()
         {
-            return pageBuilders.stream()
-                    .mapToLong(PageBuilder::getRetainedSizeInBytes)
-                    .sum();
+            long retainedSizeInBytes = 0;
```

Comment:
Can you add a comment to explain the benefit of using for loops? Same for the one below.